pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -696,6 +696,14 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 					Placeholder:  `{{ template "default.message" . }}`,
 					PropertyName: "message",
 				},
+				{ // New in 9.0.
```

Comment:
```suggestion
				{ // New in 9.1.
```

NIT: Wrong version in the comment